lib/internal/fs/recursive_watch.js

```diff
@@ -202,6 +202,8 @@ class FSWatcher extends EventEmitter {
         this.emit('change', 'rename', pathRelative(this.#rootPath, file));
       } else if (currentStats.isDirectory()) {
         this.#watchFolder(file);
+      } else {
+        this.emit('change', 'change', pathRelative(this.#rootPath, file));
```

Comment:
Can you add a test case for this? I didn’t understand why we need this.

Reply:
Sure I will add!
this is needed since this simple use-case doesn't seem to work (I might have missed something?):

does not trigger on linux, but does trigger on mac:
```js
const { watch, FSWatcher } = require('fs');

const watcher = watch(__dirname, { recursive: true }, () => {
  console.log('changed');
});
```

Reply:
What and when are you changing? It’s quite ambiguous what the issue is.

Reply:
create a file `a.js`, set its content to this ^ and run it.
then save the file is expected to print `change`. prior to my change it only worked with the native recursive watch

Reply:
Weird that we didn’t have that test case right until now. I’d have more info with a test case.

Reply:
Yep I was surprised as well